tutor/src/screens/assignment-builder/builder/tasking.js

```diff
@@ -127,79 +129,84 @@ class Tasking extends React.Component {
   }
 
   render() {
-    if (!this.taskings.length) { return null; }
+    const tasking = this.taskings.length ? this.taskings[0] : null;
```

Comment:
Seems odd that an array of taskings is passed into here but only the first one is used.

Reply:
yeah, it's a bit strange.  it's because of when they're combined it sets all the tastings for periods at once